site/profile/manifests/cvmfs.pp

```diff
@@ -123,6 +136,21 @@
   selinux::boolean { 'use_fusefs_home_dirs': }
 }
 
+# Create an alien source that refers to the uid and gid of cvmfs user
+class profile::cvmfs::alien_cache (
+  String $alien_fs_root = '/scratch',
+  String $alien_folder_name = 'cvmfs_alien_cache',
+) {
+  $uid = lookup('profile::cvmfs::local_user::uid', undef, undef, 13000004)
+  $gid = lookup('profile::cvmfs::local_user::gid', undef, undef, 8000131)
+
+  file {"/mnt/${alien_fs_root}/${alien_folder_name}":
```

Comment:
This could potentially result in Puppet trying to recreate the File resource at every run because of the double slash i.e.: `/mnt//scratch/cvmfs_alien_cache` != `/mnt/scratch/cvmfs_alien_cache` .

`$alien_fs_root` should be sanitized to remove any heading and trailing slashes.

Reply:
I am removing both trailing and leading slashes now